### ISSUE TITLE
multi-classes defects detect?

### ISSUE THREAD

**Issue by jjqcat** [vicoslab/mixed-segdec-net-comind2021#28]
Hi, this model can extend to support multi-classes defects detection?  Thanks

**Comment by JakobBozic** [comment]
I don't see any reason why it couldn't be, however, we have never tried to do it.

**Comment by jjqcat** [comment]
you're right, I have still a question the segment network output result shape is not accurate as the defect's in the origin image, because of downsampling?

**Comment by JakobBozic** [comment]
Yes, segmentation output is downsampled by a factor of 8. Also, for this reason, the input size has to be divisible by 8.

**Comment by wang-gang** [comment]
> Yes, segmentation output is downsampled by a factor of 8. Also, for this reason, the input size has to be divisible by 8.



> Hi, this model can extend to support multi-classes defects detection? Thanks

hello，sir. Have you solved the problem of multiple classifications？Can you help me？Thanks！
